src/effects/native/echoeffect.cpp

```diff
@@ -26,26 +27,28 @@ EffectManifest EchoEffect::getManifest() {
 
     EffectManifestParameter* delay = manifest.addParameter();
     delay->setId("delay_time");
-    delay->setName(QObject::tr("Delay"));
-    delay->setDescription(QObject::tr("Delay time (seconds)"));
+    delay->setName(QObject::tr("Time"));
+    delay->setDescription(QObject::tr("Delay time\n"
+        "1/4 - 2 beats rounded down to nearest 1/4 beat if sync parameter is enabled and tempo is detected (decks and samplers) \n"
+        "1/4 - 2 seconds if sync parameter is disabled or no tempo is detected (mic & aux inputs, master mix)"));
     delay->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
     delay->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
-    delay->setUnitsHint(EffectManifestParameter::UnitsHint::TIME);
-    delay->setMinimum(0.1);
-    delay->setDefault(1.0);
-    delay->setMaximum(EchoGroupState::kMaxDelaySeconds);
+    delay->setUnitsHint(EffectManifestParameter::UnitsHint::BEATS);
+    delay->setMinimum(0.0);
+    delay->setDefault(0.5);
+    delay->setMaximum(2.0);
 
     EffectManifestParameter* feedback = manifest.addParameter();
     feedback->setId("feedback_amount");
     feedback->setName(QObject::tr("Feedback"));
     feedback->setDescription(
             QObject::tr("Amount the echo fades each time it loops"));
-    feedback->setControlHint(EffectManifestParameter::ControlHint::KNOB_LOGARITHMIC);
+    feedback->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
     feedback->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
     feedback->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
-    feedback->setMinimum(0.00);
-    feedback->setDefault(0.5);
-    feedback->setMaximum(1.0);
+    feedback->setMinimum(0.25);
+    feedback->setDefault(0.75);
+    feedback->setMaximum(0.90);
```

Comment:
That seams to be a known issue of an echo/delay effect, and sometimes a desired feature. 

> Each one of these repetitions will gradually fade out as new ones are created, unless you have the feedback cranked up full, in which case the signal will keep compounding until it reaches a deafening squall.

http://www.musicradar.com/tuition/tech/the-ultimate-guide-to-effects-delay-457920

I have just read some stompbox manuals and there is no other feedback range than 0 .. 100 % implemented. 

I think we have an other issue here with our gain leveling. If we play a track with RG loudness, the digital level is likely at 50 %, our delay buffer clamper on the other hand, clamps on digital full level, which is most likely much louder than the source signal and will clip the soundcard output with any tiny other signal added. 

If we use a tanh clipper this would be better because it kicks in somhow earlier. We can also add a gain to the clipping value to let et clip even earlier and may also make this a parameter which is default to a value that matches the input level.